### ISSUE TITLE
Any chance `events` will live update?

### ISSUE THREAD

**Issue by pedsm** [estiens/world_cup_json#210]
Currently, the current game doesn't get live event updates:

```json
"home_team_events": [],
"away_team_events": [],
```

Only returning empty strings. Any chance this could be live (a minute or two delay is what I mean when I say "live")

**Comment by estiens** [comment]
I will look and see if there were any errors, they have been coming in live just fine for me though because I ingest them with a slackbot and have even seen goals and subs show up before I saw them on the TV

**Comment by dg01d** [comment]
Hi @estiens 

There's definitely noting coming through from `/matches/current` in home_team or away_team events

As pedsm says, its just an empty array.

**Comment by estiens** [comment]
Sorry ya'll. Watching USA ENG at a bar so can't debug right now but will look at it later!

**Comment by estiens** [comment]
I can restart it from here though. First debugging step anyway while pulling up the logs :D

**Comment by pedsm** [comment]
Yes sorry I did see some events come in live after I posted this.

**Comment by pedsm** [comment]
Looks to be coming through on my end (might have been because the current game had no events before @dg01d )

**Comment by estiens** [comment]
There is a possibility one of the follower databases was overloaded and stopped writing, so depending on which db you were routed to it might not have retrieved the events, should be fixed now and everything has looked again today for me

Current stats: 3.5k RPM/300ms response time with no errors

**Comment by pedsm** [comment]
Looks like the events are updating live, but I did notice a bit of an edge case. On game 28 `https://worldcupjson.net/matches/28` we had a Goal event followed by a VAR cancellation. This keeps the goal event around. Not sure what the behaviour should be there?
